turborepo-tests/e2e/tests/prune-docker.ts

```diff
@@ -0,0 +1,86 @@
+import path from "path";
+import * as assert from "uvu/assert";
+import * as uvu from "uvu";
+import {
+  getImmutableInstallForPackageManager,
+  getCommandOutputAsArray,
+  getLockfileForPackageManager,
+} from "../helpers";
+import { PackageManager } from "../types";
+import { Monorepo } from "../monorepo";
+
+export default function (
```

Comment:
Not in the scope of this PR, but I think that the `prune/*.t` tests might cover all of the behavior that gets tested here.